Themis.xcodeproj/project.pbxproj

```diff
@@ -1874,6 +1887,30 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		6D0A527C25FA2E7000C49E55 /* XCRemoteSwiftPackageReference "openssl-apple" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/julepka/openssl-apple";
+			requirement = {
+				kind = upToNextMajorVersion;
```

Comment:
Given that versioning here is mostly nonsensical, I believe it would be more reasonable to pin the “exact” version. OpenSSL *does not* use semver, the second digit is their major version and SPM will think it's the first. For example, 1.0.2 => 1.1 has introduced a lot of breakage.

It's not much of a problem given that a) users will not be affected, only maintainers, and b) we maintain and control CLOpenSSL releases as well. But if there is a way to prevent nasty surprises, I'd take it.

You can't shoehorn OpenSSL versioning scheme into semver. (Well, actually *you can*, but using 101.1.10802 for 1.1.1h-2 is hardly elegant.)